KITE-Licode-Test/js/pages/MainPage.js

```diff
@@ -58,6 +78,18 @@ class MainPage {
     return checked.result;
   }
 
+  async audioCheck(stepInfo, index) {
+    let checked; // Result of the verification
+    let i; // iteration indicator
+    let timeout = stepInfo.timeout;
+    stepInfo.numberOfParticipant = parseInt(stepInfo.numberOfParticipant); // To add the first video
+    console.warn(`Testing Audio with ${stepInfo.numberOfParticipant} participants, index`, index);
+    await TestUtils.waitForVideos(stepInfo, audios);

```

Comment:
np: shouldn't this be callign the new `waitForAudios`?
```suggestion
    await TestUtils.waitForAudios(stepInfo, audios);

```